vtrace/platforms/win32.py

```diff
@@ -2210,3 +2223,116 @@ def getCacheSyms(self):
             ret.append( (addr-self.loadbase, size, name, stype) )
         return ret
 
+    def genSymbols(self, symopts=None, symtypes=None):
+        '''
+        Generator of symbols... does not cache symbols.  This is good for PDBs of unusual size (POUSes).
+        '''
```

Comment:
This feels like a very over-complicated way to do this. Perf aside, what you've implemented is a fully-interlocked yield generator which doesn't really  benefit from the parser being in a separate thread anyway ( since he is locked down during the yields ).  Did you mean for the thread to be "buffering" responses?  If so, this would be really simple to do with a Queue object getting handed into the thread.  If you really are looking for the parser thread to be locked-out (aka, not get ahead) of the consumer, i see no reason to have him at all?